src/components/layout/auth/Blocked.jsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Grid, Typography, Button } from '@material-ui/core'
+
+function decodeQueryParam(p) {
+  return decodeURIComponent(p.replace(/\+/g, ' '))
+}
+
+export default function Blocked({ serverSettings }) {
+  const { t } = useTranslation()
+  const params = window.location.href.split('/').pop()
```

Comment:
I think ReactRouter might have a nice hook for getting params.

Reply:
It does and I've used it before however I copy/pasted this from the existing code base for consistency. 😂 I can switch it tomorrow.